chatcommunicate.py

```diff
@@ -502,11 +550,17 @@ def handle_commands(content_lower, message_parts, ev_room, ev_room_name, ev_user
             post_message_in_room(ev_room, logs_part, False)
     if content_lower.startswith("!!/pull"):
         if is_privileged(ev_room, ev_user_id, wrap2):
-            r = requests.get('https://api.github.com/repos/Charcoal-SE/SmokeDetector/git/refs/heads/master')
```

Comment:
Keep this as it was, the lines even fit in the browser page so I don't think it's too long